x-pack/solutions/security/plugins/security_solution/public/data_view_manager/components/data_view_picker/index.tsx

```diff
@@ -65,6 +65,7 @@ export const DataViewPicker = memo(({ scope, onClosePopover, disabled }: DataVie
   const savedDataViews = useSavedDataViews();
 
   const isDefaultSourcerer = scope === DataViewManagerScopeName.default;
```

Comment:
We should probably rename these at some point, but I get why they are named this way since the URL parameter is technically `sourcerer`